<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProductsInElaboratedProductAdd</class>
 <widget class="QDialog" name="frmProductsInElaboratedProductAdd">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Managing products in an elaborated product</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/caloriestracker.qrc">
    <normaloff>:/caloriestracker/books.png</normaloff>:/caloriestracker/books.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_16">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 128, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Select a product</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbProducts">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Select a format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbFormats">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="myQDoubleSpinBox" name="spnAmount">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="showGroupSeparator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> g</string>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bb">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">myqdoublespinbox</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/caloriestracker.qrc"/>
 </resources>
 <connections/>
</ui>
